apps/src/templates/InlineMarkdown.jsx

```diff
@@ -27,7 +27,8 @@ export default class InlineMarkdown extends React.Component {
   };
 
   render() {
-    const rendered = markdownToReact.processSync(this.props.markdown).result;
+    const rendered = markdownToReact.processSync(this.props.markdown.trim())
```

Comment:
I'm going to look into why it crashes... but in the meantime, it would be nice to note that the `trim` is a load bearing column and must be there to fix a bug and maybe link this PR/issue/ticket/etc haha.

Reply:
Thank you!